docs/source/common/trainer.rst

```diff
@@ -416,19 +416,19 @@ benchmark
 
 |
 
-If true enables cudnn.benchmark.
-This flag is likely to increase the speed of your system if your
-input sizes don't change. However, if it does, then it will likely
-make your system slower.
+The value (``True`` or ``False``) to set ``torch.backends.cudnn.benchmark`` to. If not specified, the value
+set in the current session will be unchanged.
+Setting this flag to ``True`` is likely to increase the speed of your system if your input sizes don't
+change. However, if it does, then it will likely make your system slower.
 
 The speedup comes from allowing the cudnn auto-tuner to find the best
 algorithm for the hardware `[see discussion here]
 <https://discuss.pytorch.org/t/what-does-torch-backends-cudnn-benchmark-do/5936>`_.
 
 Example::
 
-    # default used by the Trainer
-    trainer = Trainer(benchmark=False)
+    # default used by the Trainer (current setting for torch.backends.cudnn.benchmark is unchanged)
```

Comment:
minor nit for docs:

```suggestion
# default used by the Trainer (current setting for ``torch.backends.cudnn.benchmark`` is unchanged)
```

Reply:
I think literal blocks have to be indented in RST https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
Actually, since this comment is also in the literal example block, it is code also. So no backticks are needed if we are to follow the convention in the rest of this file

Reply:
Apologies for the indentation suggestion, it was by fault.

And sounds good, you are right!